proposals/stage-2/CODE_OF_CONDUCT/HANDLING-REPORTS.md

```diff
@@ -10,11 +10,16 @@ Generally, reports to coc-escalation should refer to a report that was already s
 
 * report@openjsf.org or
 * one of the project's reporting emails 
-* explanation of why the reportee does not feel comfortable using the previous reporting emails for the initial report
+* explanation of why the reporter does not feel comfortable using the previous reporting emails for the initial report
 
-If a report to coc-esclation does not include one of those, the reportee will be asked to to provide one of those three as the 
+If a report to coc-esclation does not include one of those, the reporter will be asked to to provide one of those three as the 
 reason for reporting to coc-escalation.
 
+## Code of Conduct
+
+The OpenJS Foundation and its member projects use the Contributor Covenant v1.4.1 as its Code of Conduct. Refer to the [Code of Conduct](https://github.com/openjs-foundation/cross-project-council/blob/master/CODE_OF_CONDUCT.md) for the full text and reporting and esclation procedures. 
```

Comment:
This might be a bit pedantic, but our `CODE_OF_CONDUCT.md` document does not in fact contain "the full text" of our code of contact; instead it links to contributor-covenant.org for the full text.

Could we add the English-language text to our own CoC document, and link to the source for the translations?

Reply:
Why would we want the english text to be saddled with a redundant source of truth, while the translations remain properly single-source-of-truth-ed?

Reply:
First of all, for ease of use and to match the expectations of the covenant text itself; it includes this near the end:
>This Code of Conduct is adapted from the Contributor Covenant, version 1.4, available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html

Second, because by referring to and trusting only an external source, we run a risk that said external source will change the text. This might sound like a theoretical risk, but the first "version 1.4.1" of the covenant [was published](https://github.com/ContributorCovenant/contributor_covenant/commit/5738a899ea6374469396536a0a7fabe5a96c7fb7) in April 2017, and since then it's been modified by [at least these commits](https://github.com/ContributorCovenant/contributor_covenant/commits/release/content/version/1/4/code-of-conduct.md). Those are not all cosmetic, as categories of characteristics have been added to the initial pledge and the scope has been amended, all without changing the version number.

Now that version 2.0 of the covenant has been published, it's of course less likely that 1.4 will be further amended, but it's not impossible either. Having the actual text under our control would allow us to be clear about what it actually says.

Reply:
Perhaps then we should link to both the web version, and the git sha?

If we're going to inline the text, then why link elsewhere at all? Your concerns apply to the translated text just the same as to the english text.

Reply:
I think the concern with having the CoC inlined was that it could get out of date and we would have to update that every time we made changes. 

I can make changes to the text inline to say "Go here to find a link to the CoC text"

Reply:
I would actually count it as a good thing for us to need to be explicit about updating our CoC when it changes. And we'd need to do that even when linking to the text, as its version number would hopefully change on updates.

But sure, this is a separate issue if the inline text here is updated to reflect the current situation.

Reply:
Added #413.